docs/CONFIG-PROPERTIES.md

```diff
@@ -70,11 +66,37 @@
 | goroutine.leak.detection.keep.stats.hours | integer (hours) | 24 | Amount of hours to keep the stats for leak detection. We keep more stats than the check window to be able to react to settings with a bigger check window via configuration. |
 | goroutine.leak.detection.cooldown.minutes | integer (minutes) | 5 | Cooldown period in minutes after the leak detection is triggered. During this period, no stack traces are collected; only warning messages are logged. |
 
-In addition, there can be per-agent settings.
-The Per-agent settings begin with "agent.*agentname*.*setting*"
-The following per-agent settings override the corresponding default ones:
+
+## Log levels
+Log level can be set for three different components of EVE: zedbox, syslog, and kernel.
+The log levels set this way are used to control the verbosity of the logs produced by the corresponding components.
+All logs produced this way will be saved locally in /persist/newlog/keepSentQueue/ directory and will be subject to rotation.
+
+Due to implementation specifics, there are two different sets of log levels that can be set: logrus and syslog levels.
+Logrus levels are used by zedbox agents, while syslog levels are used by syslog and kernel.
+
+* the logrus levels are as follows: panic, fatal, error, warning, info, debug, and trace ["https://pkg.go.dev/github.com/sirupsen/logrus"].
+* the syslog levels are as follows: emerg, alert, crit, err, warning, notice, info, debug ["https://man7.org/linux/man-pages/man3/syslog.3.html"].
+
+Additionally all log levels can be set to "none" to disable logging for the corresponding component or to "all" to enable all log levels.
+
+Furthermore, the "remote" log levels control the verbosity of the logs sent to the controller.
```

Comment:
How about adding "subset of the above logs" here? You explain that in more detail elsewhere but it might make sense to state it here that the remote can never include more that the ones generated and saved.